### PR TITLE
Update the GUI so that it is uses current terminology

### PR DESCRIPTION
**Pull request by juggernautsei**
We had a hard time finding CMS ID. The term for it now is Payer ID and the GUI should reflect that.

### CODE REVIEW

**Comment by bradymiller** [comment]
hi @juggernautsei ,
Does this also require change in the "CMS ID is missing for payer" messages in the library/gen_x12_837i.inc.php script, or is that something else?
-brady

**Comment by bradymiller** [comment]
oops, closed by mistake; just re-opened.

**Comment by juggernautsei** [comment]
No, it does not change the error message. I will add that to the PR

File: interface/code_systems/cqm_valueset_howto.php [hunk 1]
```diff
@@ -28,7 +28,7 @@
 <div class="dialog"><p>
 <?php echo xlt("Steps to install the VALUSET database"); ?>:
 <ol>
-<li><?php echo xlt("The first step is to download the VALUESET release. Access to VALUESET is provided by NLM. Only valueset for Eligible Professionals need to be downloaded and it should be downloaded in XML format from Sorted By CMS ID column. For more details see the below link") .
+<li><?php echo xlt("The first step is to download the VALUESET release. Access to VALUESET is provided by NLM. Only valueset for Eligible Professionals need to be downloaded and it should be downloaded in XML format from Sorted By Payer ID column. For more details see the below link") .
 " <a href='https://vsac.nlm.nih.gov/#download-tab'>https://vsac.nlm.nih.gov/#download-tab</a>."; ?>
```
Comment (stephenwaite) [inline interface/code_systems/cqm_valueset_howto.php]: hi @juggernautsei , the cms id here is not referencing the payer id. cms id is ok for the cqms

Comment (juggernautsei) [reply interface/code_systems/cqm_valueset_howto.php]: Ok, I have reverted that change.

**Comment by bradymiller** [comment]
hi @juggernautsei ,
Looks good and bringing this into codebase now. Thanks for the code improvement! -brady

**Pull request merged**